recipes/mp-units/config.yml

```diff
@@ -1,4 +1,6 @@
 versions:
+  "2.4.0":
+    folder: 2.4.0
```

Comment:
Thanks @mpusz !
looks like this folder doesn't exist, is this mean to be a new folder or to reuse 2.3.0?

Reply:
Sorry, my fault :-(

Reply:
Fixed!